### ISSUE TITLE
Can't edit Context Menu for current user

### ISSUE THREAD

**Issue by Anton-V-K** [BluePointLilac/ContextMenuManager#159]
The program requires elevation at start and after that it runs with another user (Administrator), so it's not possible to edit context menu entries for the current ordinary user (i.e. under `HKEY_CURRENT_USER`).

**Comment by LesFerch** [comment]
I agree. IMO, this is a serious limitation. The program should, by default, NOT require UAC.

It should start up normally and existing items, that are in HKLM, should be grayed out. Items that are already in HKCU would not be grayed out. New items should, by default, be created in HKCU. There should be a menu item to elevate via UAC (which would then allow editing of all items) and an option to select if new items are created in HKLM or HKCU.
